combat-trainer.lic

```diff
@@ -1287,6 +1358,30 @@ class SpellProcess
     end
   end
 
+  def check_trader_magic(game_state)
+    return unless DRStats.trader?
+    if Flags['ct-starlight-depleted'] then
```

Comment:
💡 We typically omit the `then` from if-blocks

Reply:
Nice catch